docs/meshcloud.customer.md

```diff
@@ -23,11 +23,21 @@ Depending on the configuration of your meshStack implementation, you may be able
 
 ## Invite users to a meshCustomer team
 
-If you would like to give others access to your meshCustomer and the related meshProjects, go to your "Account" settings. You can access them by pressing the settings icon on the top right of the meshPanel. From here, navigate to "Users" and provide the full name as well as the email address of that person. You can initially setup the role in the dropdown which describes the access level of the invited user. Press "+" to send an invitation email. This step is necessary in order to give a user access to your [projects](meshcloud.project.md). After this action the created invitation will be appear under the pending customer user role requests section.
+If you would like to give others access to your meshCustomer and the related meshProjects, go to your "Account" settings.
+You can access them by pressing the settings icon on the top right of the meshPanel.
+From here, navigate to "Users" and provide the full name as well as the email address of that person.
+You can initially setup the role in the dropdown which describes the access level of the invited user.
+Press "+" to send an invitation email. This step is necessary in order to give a user access to your [projects](meshcloud.project.md).
+After this action the created invitation will appear under the pending customer user role requests section.
 
 ## Assign meshCustomer Roles
 
-You can change the role assigned to each user on the current meshCustomer. To change the assigned role choose a new role from the dropdown and save the changes via the disc icon. The following roles are available:
+You can change the role assigned to each user on the current meshCustomer. 
+To change the assigned role choose a new role from the dropdown and save the changes via the disc icon.
+
+A user can be assigned multiple roles simultaneously. The user will recieve the combined rights of all the roles assigned.
```

Comment:
Above line is the only new part. I just added some new lines on the existing stuff so that it's easier to read in the IDE.